.github/workflows/sonarcloud.yml

```diff
@@ -85,6 +85,7 @@ jobs:
         with:
           antares-deps-version: ${{steps.antares-deps-version.outputs.prop}}
           os: ${{matrix.os}}
+          buildtype: Debug
```

Comment:
unused, consider removing

Reply:
seems that dependencies cannot be built correctly without this. cmake fails with:

```
CMake Error at CMakeLists.txt:249 (message):
  Sirius solver not found.  Sirius solver can be compiled with
  -DBUILD_sirius=ON or you can specify previous dependency install directory
  with -DCMAKE_PREFIX_PATH or -DDEPS_INSTALL_DIR
```

Reply:
you are right, sorry